CONTRIBUTING.md

```diff
@@ -0,0 +1,339 @@
+
+---
+>**Example for `<optional footer>`**:
+>:point_right:Closes #123:point_left:
+---
+
+### <a name="4.2.3.3 "></a> 4.2.3.3 Commit Examples
+:shit:
+**Bad**
+
+```console
+docs(readme): fix orthography, remove out of date paragraph and fix broken links
+```
+:+1:
+**Good**
+
+```console
+docs(readme): document design improvement change content
+
+fix orthography
+remove out of date paragraph
+fix broken links
+```
+
+### <a name="4.2.4"></a> 4.2.4 Push your Changes
+Pushing refers to **sending your committed changes to a remote repository**, such as a repository hosted on GitHub. For instance, if you change something locally, you'd want to then push those changes so that others may access them.
+
+After working on your changes you need to Push it (upload) your **newly created branch** to GitHub
+
+```console
+    git push origin created-branch
+```
+
+### <a name="4.2.5"></a> 4.2.5 Create a Pull Request
+
+Pull requests or PR are **proposed changes** to a repository submitted by a user and accepted or rejected by a repository's collaborators.
+
+After all the work being pushed to the newly created branch, In GitHub, send a pull request to our [repository.](https://github.com/flyve-mdm/glpi-plugin/pulls)
+
+### <a name="4.2.5.1"></a> 4.2.5.1 How to Write a Title for a Pull Request.
+Pull Request should be named in reference to the main fix or feature you provide, minor information can be added to the description.
+
+:straight_ruler:
+**Title Lenght:** Keep it concise and don't write more than **50 characters** in the title.
+
+:construction:
+**For Work in Progress (WIP):** If you don’t want your PR to be merged accidentally, add the word "wip" or "WIP" to its title and the [WIP bot](https://github.com/apps/wip) will set its status to error.
+
+---
+>**Example for `Titles for work in progress (WIP):`**
+>:point_right:WIP Contribution Guideline Improvement.:point_left:
+---
+
+:white_check_mark:
+**Finalized Work:** If you are done with your work and want it to be merged, just write a descriptive title with no more than 50 characters.
+
+---
+>**Example for `Titles for Finalized Work:`**
+>:point_right:Contribution Guideline Improvement.:point_left:
+---
+
+### <a name="4.2.5.2"></a> 4.2.5.2 How to Write a Description for a Pull Request.
+
+We provide a [template](https://github.com/flyve-mdm/glpi-plugin/community) for Pull Request descriptions. When you're creating a Pull Request it'll be shown automatically. Just fill it out and you're done.
+
+### <a name="4.2.5.1"></a> 4.2.5.1 Additional Information.
+
+:page_with_curl:
+**Choose the right label**: Look at the [list of available labels.](https://github.com/flyve-mdm/glpi-plugin/issues/labels)
+
+### <a name="4.2.6"></a> 4.2.6 How to proceed with suggestions.
+
+If we suggest changes then:
+* Make the required updates.
+* Re-run the Angular test suites to ensure tests are still passing.
```

Comment:
@bootbeep not only are Angular test executed, the correct sentence would be "Re-run the test suites to ensure tests are still passing".

Reply:
Got it!

Reply:
This project doesn't use Angular for testing it uses Atoum